azkaban-common/src/main/java/azkaban/user/UserUtils.java

```diff
@@ -53,8 +54,8 @@ static void setupWatch(final Map<String, ParseConfigFile> configFileMap) throws
     final WatchService watchService;
     try {
       watchService = FileSystems.getDefault().newWatchService();
-    } catch (IOException e) {
-      log.warn(" Failed to create WatchService " + e.getMessage());
+    } catch (final IOException e) {
+      log.warn(" Failed to create WatchService " + e.toString());
```

Comment:
log.warn can also take Exception as an argument. Can you please change to pass in the Exception instead of concatenating.